opentracing/tracecontext.go

```diff
@@ -49,18 +49,20 @@ type TraceContext interface {
 	// HTTP headers are case insensitive.
 	//
 	// `restrictedKey` MUST match the regular expression
-	// `(?i:[a-z0-9][-a-z0-9]*)` and is case-insensitive. See
-	// CanonicalizeTraceTagKey().
+	// `(?i:[a-z0-9][-a-z0-9]*)` and is case-insensitive. That is, it must
```

Comment:
I'm sure there's a better way to word it, but the idea was that `TraceAttribute` (or whatever we end up calling it) will do case-insensitive matching on the `restrictedKey`.